apps/researcher/src/app/[locale]/communities/page.tsx

```diff
@@ -21,6 +23,7 @@ interface Props {
     query?: string;
     sortBy?: SortBy;
     offset?: number;
+    onlyMyCommunities?: number;
```

Comment:
The name `onlyMyCommunities` suggests this is a `boolean`, not a `number`. Is there a way to either rename the property or to change the type? If not, add a comment?

Reply:
`number` is wrong. It is either the string 'true' or undefined, so I have renamed it to `onlyMyCommunities?: 'true';`